apps/lib/apps.c

```diff
@@ -200,14 +200,10 @@ unsigned long get_nameopt(void)
     return (nmflag_set) ? nmflag : XN_FLAG_ONELINE;
 }
 
-int dump_cert_text(BIO *out, X509 *x)
+void dump_cert_text(BIO *out, X509 *x)
 {
     print_name(out, "subject=", X509_get_subject_name(x));
-    BIO_puts(out, "\n");
     print_name(out, "issuer=", X509_get_issuer_name(x));
-    BIO_puts(out, "\n");
```

Comment:
Doesn't this change the format? Does print_name always terminate the output with EOL?

Reply:
Yes - as can be seen easily from its definition.
And if is only partly printed newlines it would be pretty weird.

Reply:
Well, i meant under the assumption that `out` is non-NULL :wink:
If it is NULL, of course nothing is printed at all.